contents/blog/brand.md

```diff
@@ -0,0 +1,128 @@
+
+We sometimes make business decisions based on who we are, and what we want to do - not always what may, for example, generate the most revenue growth.
+
+For example, we set out initially expecting to have an open core model. We'd offer open source software, and some people in big enterprises, would self host and pay for the party for everyone else. We felt this would work as there was no real competition, whereas in cloud, we had lots of competitors with over $100M in annual revenue. However, we simply love shipping new features, and after our first hackathon, we wound up with session replay (ie a new feature, big enough to be its own product), which took off and this resulted in a multi-product approach. As a result, we could compete in cloud by being way wider than everyone else. Eventually, we removed our paid self hosted offering entirely, went all in on multi product in cloud, focused on "getting in first" with earlier stage companies instead of the largest enterprises, and our motivation, ambition and growth all took off as a result. And yes we kept the open source product - it's more expensive though than cloud for the majority of users because you have to host it and we have big free tiers.
+
+## Brand isn't just your website...
+
+... it is instead how your company is experienced by others - consider your _company_ your product.
+
+Here is a non-exhaustive list, there are dozens more examples than this though:
+
+* it's how much you invest in each function in your company relative to each other
+* it's what you look for when you hire people, and it's the process you use to hire with
+* how you decide what to build - do you ship features early, or do you focus on meeting the needs of your largest customer with every new thing
+* how much you are design vs engineering vs product vs sales vs marketing led - how many people are in each function but also your processes for deciding what to work done
+* the frequency and style with which you email your customers, for marketing, product updates or your activation flow
+* the kind of people you hire into Customer Success or Sales
+* the background experience of those in your marketing team
+* the tone of voice on your website
+* who is on your board
+* how your pricing works
+* how your social media works
+* how helpful you are to random founders that message you
+* the marketing channels you pick
+* it's what you talk about in your blog - I'm writing this piece for brand reasons (transparency / equipping developers (with knowledge) to build successful products), nothing else - I don't care or try to generate conversions from this post, I just want to leave an impression on people. And it's who I am - I'm writing this to pass the time on a long plane journey.
+
+## You can't 80:20 everything
+
+A lot of design / brand work has an _increasing_ marginal return. This is because a great brand's purpose is that you stand out. You do _not_ stand out by being average!
```

Comment:
```suggestion
A lot of design or brand work has an _increasing_ marginal return. This is because a great brand's purpose is that you stand out. You do _not_ stand out by being average!
```